lib/spectacles.rb

```diff
@@ -40,4 +40,13 @@ def trailer(stream)
   end
 end
 
+ActiveRecord::ConnectionAdapters::SchemaStatements.class_eval do
+  alias_method(:_spectacle_drop_table, :drop_table) if method_defined?(:drop_table)
+
+  # Add this so that views get cleaned up properly
+  def drop_table(name, options={})
+    execute "DROP TABLE #{name} CASCADE"
```

Comment:
MySql allows CASCADE option for compatibility, but ignores it
SQLite3 does not allow CASCADE on DROP TABLE
MSSqlServer doesn't support CASCADE on DROP TABLE.  It requires additional code to accomplish this
Vertica provides full support for DROP TABLE ... CASCADE
Postgresql provides full support for DROP TABLE ... CASCADE

It appears that to support this functionality across all DB's spectacles supports, we will have to do something like check the DB adapter code, and see if `drop_view` exists.  If it does, and if the `:force => true` option is set, call `drop_view`.  Otherwise, call standard `drop_table`.

Does that sound reasonable?